src/styles/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $second-text-color;
```

Comment:
Ensure that the SCSS variable `$second-text-color` is defined in a separate SCSS variables file as required by the task.